src/cli/imports.rs

```diff
@@ -213,33 +141,337 @@ impl fmt::Display for Import {
+                // update column width to match longest string in column
+                for (idx, print_str) in print_vec.iter().enumerate() {
+                    col_width[idx] = cmp::max(col_width[idx], print_str.len());
+                }
+
+                dyn_imports.push(print_vec);
+            }
         }
     }
+
+    // print collected dynamic relocations
+    println!("\nOther dynamic symbols:");
+    print_imports(&headers, dyn_imports, &col_width, no_color);
 }
 
 pub fn dump_pe_imports(pe: &goblin::pe::PE, no_color: bool) {
-    println!("Imports:");
-    println!("  'name': dll [ordinal, offset, rva]");
+    // column headers for PE imports
+    let headers = vec!["Name", "DLL", "Rel Virt Addr", "Ord", "Offset"];
+    // initialize column width vec based on header length
+    let mut col_width = vec![0; headers.len()];
+    for (idx, header) in headers.iter().enumerate() {
+        col_width[idx] = header.len();
+    }
+
+    // collect imports
+    let mut imports = Vec::new();
     for import in pe.imports.iter().as_ref() {
-        println!("  {}", Import::from_pe(import, no_color));
+        let print_vec = Import::from_pe(import, no_color).get_print_vec();
+        // update column width to match longest string in column
+        for (idx, print_str) in print_vec.iter().enumerate() {
+            col_width[idx] = cmp::max(col_width[idx], print_str.len());
+        }
+
+        imports.push(print_vec);
     }
+
+    // print collected imports
+    println!("Imports:");
+    print_imports(&headers, imports, &col_width, no_color);
 }
 
 pub fn dump_macho_imports(macho: &goblin::mach::MachO, no_color: bool) {
-    println!("Imports:");
-    println!("  'name': dylib [is lazily evaluated?, offset, address, addend, is weak?, start of sequence offset]");
+    // column headers for Mach-O imports
+    let headers = vec![
+        "Name",
+        "Dylib",
+        "Address",
+        "Seq. Off.",
+        "Offset",
+        "Addend",
+        "Lazy?",
+        "Weak?",
+    ];
+    // initialize column width vec based on header length
+    let mut col_width = vec![0; headers.len()];
+    for (idx, header) in headers.iter().enumerate() {
+        col_width[idx] = header.len();
+    }
+
+    // collect imports
+    let mut imports = Vec::new();
     for import in macho.imports().expect("Error parsing imports") {
-        println!("  {}", Import::from_macho(&import, no_color));
+        let print_vec = Import::from_macho(import, no_color).get_print_vec();
+        // update column width to match longest string in column
+        for (idx, print_str) in print_vec.iter().enumerate() {
+            col_width[idx] = cmp::max(col_width[idx], print_str.len());
+        }
+
+        imports.push(print_vec);
+    }
+
+    // print collected imports
+    println!("Imports:");
+    print_imports(&headers, imports, &col_width, no_color);
+}
+
+fn print_imports(
```

Comment:
Good design for each file format's parsing function to collect/manipulate the same data and have a common printer function.